test/test-udp-multicast-join6.c

```diff
@@ -124,6 +124,9 @@ TEST_IMPL(udp_multicast_join6) {
 #else
   r = uv_udp_set_membership(&client, "ff02::1", NULL, UV_JOIN_GROUP);
 #endif
+  if(r == UV_ENODEV) 
+    RETURN_SKIP("No ipv6 multicast route");
```

Comment:
Can you add a MAKE_VALGRIND_HAPPY(); here, before the RETURN_SKIP? Otherwise the loop won't be properly closed in the test.

Other tests using this don't need to do it, because they check for IPv6 support before creating the loop or any handles.